service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/C2PbaPaymentNotTakenEventHandlerTest.java

```diff
@@ -34,8 +30,8 @@
 @SpringBootTest(classes = {C2PbaPaymentNotTakenEventHandler.class, LookupTestConfig.class,
     HmctsAdminNotificationHandler.class})
 class C2PbaPaymentNotTakenEventHandlerTest {
-    private final Map<String, Object> c2PaymentNotTakenParameters = ImmutableMap.<String, Object>builder()
-        .put("caseUrl", "null/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345")
+    private final BaseCaseNotifyData c2PaymentNotTakenParameters = BaseCaseNotifyData.builder()
+        .caseUrl("http://fpl/case/12345")
         .build();
 
     @MockBean
```

Comment:
What is idam being used for in this test?

Reply:
And the inbox lookup service.

Reply:
Also, with everything being mocked do you think there is a benefit to making this test not dependant on the sprint extension but just on the mockito extension and we can create the `C2PbaPaymentNotTakenEventHandler` in a before each method.